index.js

```diff
@@ -129,8 +179,16 @@ function filterByGenre() {}
       }
     ];
  */
-function getAllMoviesReleasedAtOrBeforeYear() {}
-
+function getAllMoviesReleasedAtOrBeforeYear(movies, year) {
+  let yearArr = []
+  for (let movie of movies){
+    if (movie.released <= year){
```

Comment:
Close!  Take a look at what exactly `movie.released` is and try to refactor your conditional.